### ISSUE TITLE
Incorrect Username or Password

### ISSUE THREAD

**Issue by sevypannella** [Codiad/Codiad#831]
I get this error after any clean installation on my CentOS 6.6 x64 with PHP 5.3.3

I've reinstalled Codiad between 15 to 20 times trying any kind of workaround:
1) Disabling APC cache
2) Bypassing the sha1|md5 encryption for the password in /components/user/class.user.php
3) Deleting several times all files in /data
4) Trying different and longer username and password
5) Disabling any php extension
7) Removing all the php session files from /var/lib/php/session
6) And other 10-15 tries so on...

but I had absolutely no luck.

Every time i reinstall it from git Codiad lets me create the 1st project, but when I try to login after such creation it states ALWAYS "Incorrect Username or Password".

It's 3 days I'm trying to find a fix for this but no way.

**Comment by daeks** [comment]
Hmm, hard to say. Could you check your browers console to see what cames back from the request?

**Comment by sevypannella** [comment]
I've found the problem: looks like it works only with php until 4.3.x while stops working with php 4.4.x or newer.

**Comment by daeks** [comment]
I currently cant test it with 4.4.x cause I am using 5.6.x

**Comment by sevypannella** [comment]
Agree: I tested on one of my customers' vps with a 5.5.x and it works smoothly.
It has problems with 4.4.x for sure. Can't say which problem tho...

**Comment by beli3ver** [comment]
Maybe you coudl write it in the Readme, so everybody now it.

**Comment by daeks** [comment]
It would be more interesing to fix this, then to add it to the readme ;-) 
Is there any error thrown in the browser console during the login? Please check the http requests and their response.

**Comment by lilltiger** [comment]
This is not a minor bug, seems quite major because php7 has the same issue.

The issue is that the config.php gets emptied and the user.php etc in the data folder also is compleatly empty.

But I do not know why it does that, still makes me unable to install Codiad on my php7 server.

**Comment by zenepay** [comment]
I got the same issue. Cannot login after setup. I am using php 5.4

**Comment by sokolik2135** [comment]
I've got this issue recently and that's what I see in console:
`Uncaught TypeError: Cannot read property 'resize' of undefined at system.js:5 at dispatch (jquery.min.js:3) at i (jquery.min.js:3)`
Tried on PHP 5.6 and PHP7
